lib/matplotlib/ticker.py

```diff
@@ -1933,14 +2031,22 @@ def view_limits(self, vmin, vmax):
 
 class SymmetricalLogLocator(Locator):
     """
-    Determine the tick locations for log axes
+    Determine the tick locations for symmetric log axes
     """
 
-    def __init__(self, transform, subs=None):
+    def __init__(self, transform=None, subs=None, linthresh=None, base=None):
         """
         place ticks on the location= base**i*subs[j]
         """
-        self._transform = transform
+        if transform is not None:
+            self._base = transform.base
+            self._linthresh = transform.linthresh
+        elif linthresh is not None and base is not None:
+            self._base = base
+            self._linthresh = linthresh
+        else:
+            raise ValueError("Either transform or linthresh "
+                             "and base must be provided.")
```

Comment:
This needs a small clarification because it's not in the docstring and it can be interpreted as either "(transform or linthresh) and base" or "transform or (linthresh and base)".